lua/gatekeeper/staticlib.lua

```diff
@@ -205,6 +205,11 @@ enum log_levels {
 	RTE_LOG_DEBUG = 8U,   /* Debug-level messages. */
 };
 
+enum log_types {
+	/* Corresponding to the values in rte_log.h. */
+	RTE_LOGTYPE_USER1 = 24U, /* User-defined log type 1. */
```

Comment:
Where's this constant needed? If it is needed, why shouldn't we use `BLOCK_LOGTYPE` instead?